slsDetectorSoftware/src/Detector.cpp

```diff
@@ -250,14 +250,14 @@ void Detector::setTimingMode(defs::timingMode value, Positions pos) {
 void Detector::acquire() { pimpl->acquire(); }
 
 void Detector::startAcquisition() {
-    if (getUseReceiverFlag({}).squash())
+    if (getUseReceiverFlag({}).squash(true))
```

Comment:
Do you need the {}  in the function call?

Reply:
No. Good catch. Removed them.